apps/web/src/services/analytics/Analytics.tsx

```diff
@@ -0,0 +1,23 @@
+import { GA_TRACKING_ID, IS_PRODUCTION, SAFE_APPS_GA_TRACKING_ID } from '@/config/constants'
+import { GoogleAnalytics } from '@next/third-parties/google'
+import { useEffect } from 'react'
+
+const Analytics = () => {
+  useEffect(() => {
+    // This needs to be added once in order for events with send_to: SAFE_APPS_GA_TRACKING_ID to work
+    window.gtag?.('config', SAFE_APPS_GA_TRACKING_ID, { debug_mode: !IS_PRODUCTION })
+
+    window.gtag?.('consent', 'default', {
```

Comment:
We initialize it with these default settings and never change them. The previous implementation was upgrading them when a cookie consent is given (and we were also deleting cookies if the consent is withdrawn).

I think it's OK for the time being but it would be good to have a full feature parity with the previous implementation.

Reply:
This is still being done inside the `useGtm` hook. All I did here is move the needed logic out of the `TagManager` to keep full parity

Reply:
We should probably rename Gtm to Ga to make it more clear